.NET/Microsoft.Recognizers.Text.DateTime/English/Extractors/EnglishMergedExtractorConfiguration.cs

```diff
@@ -16,8 +16,9 @@ public class EnglishMergedExtractorConfiguration : BaseDateTimeOptionsConfigurat
         public static readonly Regex AfterRegex =
             new Regex(DateTimeDefinitions.AfterRegex, RegexFlags);
 
-        public static readonly Regex SinceRegex =
-            new Regex(DateTimeDefinitions.SinceRegex, RegexFlags);
+        // used in Experimental mode
+        public static readonly Regex SinceRegexExp =
```

Comment:
Can you place the two declarations close to each other and have the decision to which one to assign explicitly show the choices? This will help readability.

Reply:
moved declarations close to each other.